interface/forms/dictation/report.php

```diff
@@ -18,8 +39,8 @@ function dictation_report( $pid, $encounter, $cols, $id) {
         $value = "yes";
       }
       $key=ucwords(str_replace("_"," ",$key));
-      print "<td><span class=bold>" . xlt($key) . ": </span><span class=text>" .
-        nl2br(text($value)) . "</span></td>";
+      print "<td><span class='bold'>" . xlt($key) . ": </span><span class='text'>" .
+        html_entity_decode($value, ENT_QUOTES | ENT_HTML5) . "</span></td>";
```

Comment:
This still doesn't sound like it will protect from xss. Let me look into this a bit.